microsoft-edge/webview2/release-notes/index.md

```diff
@@ -16,18 +16,309 @@ These Release Notes provide information about new features and bug fixes that ar
+  * [ICoreWebView2FindOptions::get_ShouldMatchWord](/microsoft-edge/webview2/reference/win32/icorewebview2findoptions?view=webview2-1.0.3650.58&preserve-view=true#get_shouldmatchword)
+  * [ICoreWebView2FindOptions::get_SuppressDefaultFindDialog](/microsoft-edge/webview2/reference/win32/icorewebview2findoptions?view=webview2-1.0.3650.58&preserve-view=true#get_suppressdefaultfinddialog)
+  * [ICoreWebView2FindOptions::put_FindTerm](/microsoft-edge/webview2/reference/win32/icorewebview2findoptions?view=webview2-1.0.3650.58&preserve-view=true#put_findterm)
+  * [ICoreWebView2FindOptions::put_IsCaseSensitive](/microsoft-edge/webview2/reference/win32/icorewebview2findoptions?view=webview2-1.0.3650.58&preserve-view=true#put_iscasesensitive)
+  * [ICoreWebView2FindOptions::put_ShouldHighlightAllMatches](/microsoft-edge/webview2/reference/win32/icorewebview2findoptions?view=webview2-1.0.3650.58&preserve-view=true#put_shouldhighlightallmatches)
+  * [ICoreWebView2FindOptions::put_ShouldMatchWord](/microsoft-edge/webview2/reference/win32/icorewebview2findoptions?view=webview2-1.0.3650.58&preserve-view=true#put_shouldmatchword)
+  * [ICoreWebView2FindOptions::put_SuppressDefaultFindDialog](/microsoft-edge/webview2/reference/win32/icorewebview2findoptions?view=webview2-1.0.3650.58&preserve-view=true#put_suppressdefaultfinddialog)
+
+* [ICoreWebView2FindStartCompletedHandler](/microsoft-edge/webview2/reference/win32/icorewebview2findstartcompletedhandler?view=webview2-1.0.3650.58&preserve-view=true)
+
+---
+
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+This Release SDK includes the following bug fixes.
+
+
+<!-- ---------- -->
+###### Runtime-only
+
+* Fixed local network access triggering a permission alert pop-up window.<!-- todo: already reported in Oct 6 & Oct 7 sections.  orig draft from script: Hiding local access network alert pop up -->
+
+<!-- end of Dec 2025 Release SDK -->
+
+
+<!-- Dec 2025 Prerelease SDK -->
+<!-- ====================================================================== -->
+## 1.0.3712-prerelease
+
+Release Date: December 2, 2025
+
+[NuGet package for WebView2 SDK 1.0.3712-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.3712-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 144.0.3712.0 or higher.
+
+
+<!-- ------------------------------ -->
+#### General changes
+<!-- omit section if empty; usually empty -->
+
+
+<!-- ------------------------------ -->
+#### Experimental APIs
+
+The following APIs are in Phase 1: Experimental in Prerelease, and have been added in this Prerelease SDK.
+
+
+<!-- ---------- -->
+###### Allowed port range, transport protocol
+<!-- todo: 2 separate sections? -->
+
+Allowed port range.  Also Transport Protocol.<!-- todo: lead-in -->
```

Comment:
need lead-ins for Phase 1 section(s) **Allowed port range** & **Transport protocol**

   * **1.0.3712-prerelease**
      * `/webview2/release-notes/index.md`
      * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/index?branch=pr-en-us-3635#103712-prerelease
      * External preview: https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/wv2-relnotes-dec-2025/microsoft-edge/webview2/release-notes/index.md#103712-prerelease
      * Planned live: https://learn.microsoft.com/microsoft-edge/webview2/release-notes/#103712-prerelease
      * New section.

Reply:
entered fresh comment asking for lead-in & heading